src/lib/orion_websocket/ws.cpp

```diff
@@ -259,6 +259,7 @@ orion_websocket *orion_websocket_new(RestService *serv)
     0,0,0,0,0,0,0,0,0,0
   };
 
+  lws_set_log_level(0, NULL);
```

Comment:
lws_set_log_level() remove all LWS output, now I have other problems with make ft, but it must allow to make ft work without problems